### ISSUE TITLE
auto detector Android not working

### ISSUE THREAD

**Issue by mrqaidi** [ishaquehassan/document_scanner_flutter#29]
edge detector work fine on ios  but not working at all on android    i tried on real device also same

**Comment by MarlonJD** [comment]
Same issue from me. I'm trying to find out the issue

**Comment by MarlonJD** [comment]
> > Same issue from me. I'm trying to find out the issue
> 
> no luck yet . what about you ?

Nope, i tried different Gradle versions, shrink false, true, minify true, false. Now luck yet. Another library with opencv works but not works this library :(

**Comment by MarlonJD** [comment]
https://pub.dev/packages/document_scanner this package works but, there is no multi page and pdf creation on this package. This library it better i think. İ cannot create manually take a photo in that lib.

**Comment by Tabarek** [comment]
The auto detector not working on android for both the camera and gallery!!

**Comment by CikaTom** [comment]
Is anyone looking into this issue?
